app/dashboard/views.py

```diff
@@ -363,112 +364,52 @@ def new_interest(request, bounty_id):
-                                "auth_data": f'{funder_profile.user.id}',
-                                "auth_service": "gitcoin",
-                                "locale": "en",
-                                "props": {},
-                                "notify_props": {
-                                    "email": "false",
-                                    "push": "mention",
-                                    "desktop": "all",
-                                    "desktop_sound": "true",
-                                    "mention_keys": f'{funder_profile.handle}, @{funder_profile.handle}',
-                                    "channel": "true",
-                                    "first_name": "false"
-                                },
-                            }, {
-                                "tid": settings.GITCOIN_HACK_CHAT_TEAM_ID
-                            }
-                        ])
-
-                        chat_profile_interest_user = result.get()
-                        if 'message' in chat_profile_interest_user:
-                            raise ValueError(chat_profile_interest_user['message'])
-
-                        funder_profile.chat_id = chat_profile_interest_user['id']
-                        funder_profile.save()
-
-                    if profile.chat_id is None:
-                        result = create_user.apply_async(args=[{
-                            "email": profile.user.email,
-                            "username": profile.handle,
-                            "first_name": profile.user.first_name,
-                            "last_name": profile.user.last_name,
-                            "nickname": profile.handle,
-                            "auth_data": f'{profile.user.id}',
-                            "auth_service": "gitcoin",
-                            "locale": "en",
-                            "props": {},
-                            "notify_props": {
-                                "email": "false",
-                                "push": "mention",
-                                "desktop": "all",
-                                "desktop_sound": "true",
-                                "mention_keys": f'{profile.handle}, @{profile.handle}',
-                                "channel": "true",
-                                "first_name": "false"
-                            },
-                        }, {
-                            "tid": settings.GITCOIN_HACK_CHAT_TEAM_ID
-                        }]
-                        )
-
-                        chat_profile_interest_user = result.get()
-                        if 'message' in chat_profile_interest_user:
-                            raise ValueError(chat_profile_interest_user['message'])
-
-                        profile.chat_id = chat_profile_interest_user['id']
-                        profile.save()
+                    if funder_profile.chat_id is '':
```

Comment:
many times the ones creating bounties are internals gitcoin members example https://gitcoin.co/issue/wearekickback/app/201/3765 that will create a channel with vivek and the contributor.

Reply:
maybe we need some preconfiguration on this so we can setup the users from the bounty side that will make part of that channel? bounty_channel_sponsors_usernames=[]

Reply:
ok as Alex confirmed in that situations they will manage to add manually the funder